### ISSUE TITLE
Alert close button does not have discernible text

### ISSUE THREAD

**Issue by kennethlarsen** [TryGhost/Ghost#8989]
### Issue Summary

All browsers/OS: Alert close button does not have discernible text. This provides a bad experience in terms of accessibility.

Some screen readers could have a hard time telling the user that the close button is actually a close button. aXe cli tool reports this as an error as well.

### Steps to Reproduce

N/A

### Technical details:

N/A

**Comment by kevinansfield** [comment]
@kennethlarsen it looks like you've implemented a fix for this but unfortunately I can't see it as the commit only has a sub-module reference update.

If you'd like to tackle this issue please raise a PR against the [Ghost-Admin](https://github.com/TryGhost/Ghost-Admin) project - there are details about how to work with the admin client in our [contributing guide](https://docs.ghost.org/docs/working-with-the-admin-client) 😄

**Comment by rick-schubert** [comment]
This sounds like an easy task for a beginner - I would like to have a go at it! Would you mind telling me how I can cause an alert so that I can start diving into it? cc @kevinansfield @kennethlarsen
